services/121-service/src/user/user.controller.ts

```diff
@@ -280,12 +280,15 @@ export class UserController {
     description: 'No user detectable from cookie or no cookie present',
   })
   public async findMe(@Req() req): Promise<UserRO> {
-    const username = req.user.username;
-    if (!username) {
+    if (!req.user || !req.user.username) {
```

Comment:
Perhaps? (Not sure if it works)
```suggestion
    if (!req.user?.username) {
```

Reply:
@elwinschmitz We are merging anyway